app/subscriber/src/features/settings/MyMinisterSettings.tsx

```diff
@@ -13,7 +13,9 @@ export const MyMinisterSettings: React.FC = () => {
   const [, store] = useAppStore();
   const api = useUsers();
 
-  const options = ministers.map((m) => new OptionItem(`${m.name} | ${m.description}`, m.name));
+  let activeMinisters = ministers
+    .filter((m) => m.isEnabled)
+    .sort((a, b) => a.name.localeCompare(b.name));
 
```

Comment:
I feel like maybe we should modify the MinisterModel so that it has a first name and last name field.  name is then set as the concat of those two on save.  This could lead to a couple of other improvements:
- sort by last name
- the last name could also be used as an automatic alias as well

Reply:
Automatic alias isn't ideal regrettably, due to common names.  It's better to force the admin to choose the appropriate aliases to limit false positives.
Splitting first and last name could be useful for sorting, but isn't critical presently.